unit_tests/test_configuration.py

```diff
@@ -34,6 +36,46 @@ def test_bootstrap(self):
             FileContains(matcher=Contains("<numExecutors>1</numExecutors>")))
         self.assertEqual({8080}, self.fakes.juju.ports["TCP"])
 
+    def test_set_prefix(self):
+        test_file = tempfile.NamedTemporaryFile()
```

Comment:
This file will be left around after the test suite completes. Please use something like:

```
from fixtures import TempDir

# ...

    test_file = self.useFixture(TempDir()).join("test-file")
```

See the [fixtures](https://pypi.python.org/pypi/fixtures) package for more details.